packages/plugin-pack/sources/packUtils.ts

```diff
@@ -69,8 +68,8 @@ export async function genPackStream(workspace: Workspace, files?: Array<Portable
 
   process.nextTick(async () => {
     for (const file of files!) {
-      const source = portablePathUtils.resolve(workspace.cwd, file);
-      const dest = portablePathUtils.join(`package` as PortablePath, file);
+      const source = ppath.resolve(workspace.cwd, file);
+      const dest = ppath.join(toFilename(`package`), file);
```

Comment:
Shouldn't it be a `PortablePath`?
(not sure though)

Reply:
it's technically also a filename

I replaced this as part of a big "replace `as PortablePath` with `toFilename` where possible" commit, this change wasn't really intentional. I'll change it back